prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/hours-operation-page/hours-operation-page.component.ts

```diff
@@ -112,6 +116,12 @@ export class HoursOperationPageComponent extends AbstractHealthAuthoritySiteRegi
     }
   }
 
+  public onSite247(change: MatCheckboxChange): void {
+    this.formState.businessDays.controls.forEach(businessDay => {
```

Comment:
Remember we want to make sure to have typing where ever we can. So if we type businessDay here we can then remove the casting we do on the next line.